Account/FirefoxAccount.swift

```diff
@@ -242,7 +242,7 @@ open class FirefoxAccount {
             }
             
             func downloadAvatar() {
-                SDWebImageManager.shared().downloadImage(with: url, options: [SDWebImageOptions.continueInBackground, SDWebImageOptions.lowPriority], progress: nil) { (image, error, cacheType, success, url) in
+                SDWebImageManager.shared().loadImage(with: url, options: [SDWebImageOptions.continueInBackground, SDWebImageOptions.lowPriority], progress: nil) { (image, _, error, _, success, _) in
```

Comment:
Nit: use `.continueInBackground` instead of `SDWebImageOptions.continueInBackground` etc.